apis/fluentd/v1alpha1/helper.go

```diff
@@ -87,11 +88,15 @@ func (pgr *PluginResources) BuildCfgRouter(cfg Renderer) (*fluentdRouter.Route,
 		},
 	}
 
+	NonKubernetesFormat := cfg.GetNonKubernetesFormat()
```

Comment:
Local variables are not recommended to start with a capital letter

Reply:
fixed